index.bs

```diff
@@ -3194,7 +3194,9 @@ associated with or [=scoped=] to, respectively.
 
         When [=clients=], [=client platforms=], or [=authenticators=] display a {{PublicKeyCredentialEntity/name}}'s value, they should always use UI elements to provide a clear boundary around the displayed value, and not allow overflow into other elements [[css-overflow-3]].
 
-        Authenticators MAY truncate a {{PublicKeyCredentialEntity/name}} member's value so that it fits within 64 bytes, if the authenticator stores the value. See [[#sctn-strings-truncation]] about truncation and other considerations.
+        When storing a {{PublicKeyCredentialEntity/name}} member's value,
+        [=authenticators=] MUST support any value whose length in the UTF-8 character encoding is 64 bytes or less.
```

Comment:
"support" seems like an imprecise verb here. ("Sure, I support it&mdash;by truncating to 16 bytes.)

Reply:
Agreed. Rephrased as below, is that better?

>When storing a [name](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/2017.html#dom-publickeycredentialentity-name) member’s value, [authenticators](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/2017.html#authenticator) MAY truncate the value as described in [§ 6.4.1 String Truncation](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/2017.html#sctn-strings-truncation) using a size limit greater than or equal to 64 bytes.